src/components/VersionSelector/VersionSelector.tsx

```diff
@@ -43,8 +59,43 @@ const VersionSelectorComponent = ({
     return setOpen(false);
   };
 
+  const handleFocusChange = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    const { key, shiftKey } = event;
+
+    if (key === 'ArrowDown' || (key === 'Tab' && !shiftKey)) {
```

Comment:
I'm not against keeping this, but I think the tabs both directions already work!

Reply:
But let's keep, because you have an interesting edge case I think maybe at the bottom.

Reply:
I added the tab key here because we want to increment the `focusedIdx` in parallel. If we do not track it, the focused state will be out of sync